### PR TITLE
konsave: add page

### PR DESCRIPTION
**Pull request by harizms**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 2.1.2

### CODE REVIEW

File: pages/linux/konsave.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+`konsave --remove <profile name>`
+
+- Export a profile as a ".knsv":
+
+`konsave --export-profile <profile name>`
+
+
+- Import a ".knsv" file:
+
+`konsave --import-profile <path to the file>`
```
Comment (CleanMachine1) [inline pages/linux/konsave.md:34]:
```suggestion
`konsave --import-profile {{path/to/profile.knsv}}`
```

**Review by CleanMachine1 — CHANGES_REQUESTED** [review]
Thanks for your contribution!

File: pages/linux/konsave.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# konsave
+
+> A CLI program that will let you save and apply your Linux customizations with just one command!
```
Comment (kbdharun) [inline pages/linux/konsave.md:3]:
```suggestion
> A CLI program that lets you save and apply your Linux customizations with just one command!
```

File: pages/linux/konsave.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+> More information: <https://github.com/Prayag2/konsave>.
+
+
+- Save current configuration as a profile:
```
Comment (kbdharun) [inline pages/linux/konsave.md:7]:
```suggestion
- Save the current configuration as a profile:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/linux/konsave.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+- List all profiles:
+
+`konsave --list`
+
+- Remove a profile:
+
+`konsave --remove <profile name>`
+
+- Export a profile as a ".knsv":
```
[the remark this answers is not in the document]
Comment (kbdharun) [reply pages/linux/konsave.md:27]: image

Checked the code, seems like it is exported to the home directory.

[the remark this answers is not in the document]
Comment (CleanMachine1) [reply pages/linux/konsave.md:27]: Then my suggestion has been edited to include this.

File: pages/linux/konsave.md (hunk 2 above)
Comment (CleanMachine1) [reply pages/linux/konsave.md:3]: The exclamation point is not required.

**Comment by harizms** [comment]
[konsave.md](https://github.com/tldr-pages/tldr/files/10426859/konsave.md)

Made changes with your suggestions .
Should i change the <path to the  file> to {{path to the file}},
Are there any more changes needed  to be made

File: pages/linux/konsave.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# konsave
+
+> A CLI program that lets you save and apply your Linux customizations with just one command!
```
[the remark this answers is not in the document]
Comment (blueskyson) [reply pages/linux/konsave.md]:
```suggestion
> Save and apply your Linux customizations.
```

File: pages/linux/konsave.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+`konsave --remove {{profile_name}}`
+
+- Export a profile as a `.knsv` to the home directory:
+
+`konsave --export-profile {{profile_name}}`
+
+- Import a `.knsv` profile:
+
+`konsave --import-profile {{path to the file}}`
```
Comment (blueskyson) [inline pages/linux/konsave.md:32]: I'm not sure if it should be `profile_name.knsv`. Feel free to replace it base on your knowledge.
```suggestion
`konsave --import-profile {{path/to/profile_name.knsv}}`
```

Comment (kbdharun) [reply pages/linux/konsave.md:32]: Why not `profile.knsv`?

Comment (blueskyson) [reply pages/linux/konsave.md:32]: Because all other examples use `profile_name` as the placeholder.

Comment (harizms) [reply pages/linux/konsave.md:32]: So Should I Change it ?

Comment (kbdharun) [reply pages/linux/konsave.md:32]: > So Should I Change it ?

Yeah

**Review by CleanMachine1 — APPROVED** [review]
Sorry that this has been sitting for a couple weeks, thanks for your contribution!

**Pull request merged**